src/main/src/main/java/org/geoserver/security/GeoServerSecurityManager.java

```diff
@@ -2212,14 +2212,6 @@ boolean migrateFrom21() throws Exception{
         }
         
         LOGGER.info("Start security migration");
-        
-        //create required directories
-        get("security/role").dir();
-        get("security/usergroup").dir();
-        get("security/auth").dir();
-        get("security/pwpolicy").dir();
-        get("security/filter").dir();
-        get("security/masterpw").dir();
```

Comment:
I understand not needing to force directory creation, not sure how to write, or if a test case is needed